packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/VSCode/index.tsx

```diff
@@ -28,7 +28,18 @@ function VSCode({ value, language, onChange }: Props) {
     const styles = getComputedStyle(document.documentElement);
     function getColor(property: string) {
       // Weird chrome bug, returns " #ffffff " instead of "#ffffff", see: https://github.com/cloud-annotations/docusaurus-openapi/issues/144
-      return styles.getPropertyValue(property).trim();
+      const color = styles.getPropertyValue(property).trim();
+      if (color.length === 4) {
+        // change hex short codes like "#fff" to "#ffffff"
+        // to fix: https://github.com/cloud-annotations/docusaurus-openapi/issues/183
+        let res = "#"; // prepend #
+        color
+          .slice(1)
+          .split("")
+          .map((c) => (res += c + c)); // duplicate each character
```

Comment:
It feels a bit odd to modify `res` from map, because map is expected to return something. Could we change this to a for-of  loop, something like:

```js
let res = "#";
for (const c of color.substring(1)) {
  res += c + c;
}
return res;
```

Reply:
Yes, it's easier to understand with a simple for loop